### PR TITLE
Add new troubleshooting section about data plane proxy CA check

### PR DESCRIPTION
**Pull request by ereslibre**
Depends on https://github.com/linkerd/linkerd2/pull/3524

### CODE REVIEW

File: linkerd.io/content/2/tasks/troubleshooting.md [hunk 1]
```diff
@@ -834,3 +834,19 @@ See the page on [Upgrading Linkerd](/2/upgrade/).
 ```
 
 See the page on [Upgrading Linkerd](/2/upgrade/).
+
+### √ data plane proxies certificate match the CA in configuration {#l5d-data-plane-proxies-certificate-match-ca}
```
Comment (wmorgan) [inline linkerd.io/content/2/tasks/troubleshooting.md]: Wordsmithing: I suggest replacing "the CA in configuration" with "the CA's certificates"

File: linkerd.io/content/2/tasks/troubleshooting.md [hunk 2]
```diff
@@ -834,3 +834,19 @@ See the page on [Upgrading Linkerd](/2/upgrade/).
...
+
+If the trust anchor has changed while data plane proxies existed, they
```
Comment (wmorgan) [inline linkerd.io/content/2/tasks/troubleshooting.md]: Wordsmithing: I suggest replacing "existed" with "were running"

File: linkerd.io/content/2/tasks/troubleshooting.md [hunk 3]
```diff
@@ -834,3 +834,19 @@ See the page on [Upgrading Linkerd](/2/upgrade/).
...
+will need to be restarted in order to refresh this information and use
+the latest trust anchor from the Linkerd configuration.
+
+This can happen if, for example, while you had proxies deployed on
+your cluster you uninstalled Linkerd and installed it again, what
```
Comment (wmorgan) [inline linkerd.io/content/2/tasks/troubleshooting.md]: Wordsmithing: I suggest replacing ""what generated a new trust anchor Linkerd-wise" with "which created a new trust anchor in the control plane"

**Comment by ereslibre** [comment]
Thanks for the review, updated!

**Comment by ihcsim** [comment]
Thanks for the docs! Mind if we update this new section to follow the same layout as the rest of the document, by starting it with an _Example failure_?

Also, I think the output format has changed in the code, where you prepend, instead of append, the namespace.

```diff
diff --git a/linkerd.io/content/2/tasks/troubleshooting.md b/linkerd.io/content/2/tasks/troubleshooting.md
index 0fbc2cd..6f21265 100644
--- a/linkerd.io/content/2/tasks/troubleshooting.md
+++ b/linkerd.io/content/2/tasks/troubleshooting.md
@@ -837,6 +837,13 @@ See the page on [Upgrading Linkerd](/2/upgrade/).
 
 ### √ data plane proxies certificate match the CA's certificates {#l5d-data-plane-proxies-certificate-match-ca}
 
+Example failure:
+
+```bash
+‼ data plane proxies certificate match CA
+    The following pods have old proxy certificate information; please, restart them: web-559684dbc5-mb44s (emojivoto)
+```
+
 If the trust anchor has changed while data plane proxies were running, they
 will need to be restarted in order to refresh this information and use
 the latest trust anchor from the Linkerd configuration.
@@ -846,7 +853,3 @@ your cluster you uninstalled Linkerd and installed it again, which
 created a new trust anchor in the control plane, but the already existing
 meshed pods didn't restart on their own, keeping old trust anchor information.
 
-```bash
-‼ data plane proxies certificate match CA
-    The following pods have old proxy certificate information; please, restart them: web-559684dbc5-mb44s (emojivoto)
-```
```

**Comment by ereslibre** [comment]
@ihcsim thank you! Yes, I'm going to push an update in minutes so they match the style and the new error format (both on namespace location and proposed output in https://github.com/linkerd/linkerd2/pull/3524#issuecomment-542311651)

**Comment by ereslibre** [comment]
PR updated.

**Pull request merged**